spec/controllers/exports_controller_spec.rb

```diff
@@ -120,8 +121,8 @@
       end
 
       it "returns a CSV of all of the exports" do
-        header = Export::ActivitiesLevelB::Row.new(fund, nil).to_a
-        expect(CSV.parse(response.body.delete_prefix("\uFEFF")).first).to match_array(header)
+        header = CSV.parse(response.body.delete_prefix("\uFEFF")).first
+        expect(header.length).to be > 1
```

Comment:
This is a smoke test that returns *some* fields. We don't really care what they are here, because caring would cost now the headers are no longer fixed.